.github/workflows/ubuntu.yml

```diff
@@ -114,6 +114,15 @@ jobs:
         path: 'simtest.json'
         prop_path: 'version'
 
+    - name: Run parallel tests
+      if: ${{ env.IS_PUSH == 'true' }}
+      uses: ./.github/workflows/run-tests
+      with:
+        simtest-tag: ${{steps.simtest-version.outputs.prop}}
+        batch-name: short-tests
+        os: ${{ matrix.test-platform }}
+        variant: "parallel"
```

Comment:
is variant is used by ./.github/workflows/run-tests?

Reply:
instead of variant create parallel input:
parallel:
    description: 'launch test in parallel'
    required: false
    default: 1

Reply:
then ctest -C Release --output-on-failure -R ${{inputs.variant}} -j ${{inputs.parallel}}